metrics/Makefile

```diff
@@ -8,7 +8,9 @@ RELEASE_PATH ?= ../strimzi-$(RELEASE_VERSION)/examples/$(PROJECT_NAME)
 release:
 	mkdir -p $(RELEASE_PATH)
 	mkdir -p $(RELEASE_PATH)/grafana-dashboards
+	mkdir -p $(RELEASE_PATH)/prometheus-rules
```

Comment:
TBH, I'm not sure what would I understand under _prometheus rules_. Wouldn't having alerts in the name be better?

Reply:
Yeah. I renamed it. I noticed that the Prometheus configuration allows to specify more rules files, so for now we just have an alerting rules file mounted from the corresponding ConfigMap as volume. We can add more rules (recording rules as well) in the future eventually.